src/GeneralStateTestsFiller/stReturnDataTest/returndatacopy_following_too_big_transferFiller.json

```diff
@@ -0,0 +1,70 @@
+{
+    "returndatacopy_following_too_big_transfer" : {
+	"_info" : {
+	    "comment" : "This test tries RETURNDATACOPY with a non-zero size after a CALL that fails because of insufficient balance."
+	},
+        "env" : {
+            "currentCoinbase" : "0x2adc25665018aa1fe0e6bc666dac8fc2697ff9ba",
+            "currentDifficulty" : "0x20000",
+            "currentGasLimit" : "0x0a00000000",
+            "currentNumber" : "0x01",
+            "currentTimestamp" : "0x03e8",
+            "previousHash" : "0x5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6"
+        },
+        "expect" : [
+            {   "indexes" : {
+                   "data" : -1,
+                   "gas" : -1,
+                   "value" : -1
+                },
+                "network" : ["EIP150", "EIP158", "Frontier", "Homestead", "Byzantium"],
+                "result" : {
+                    "0x0f572e5295c57f15886f9b263e2f6d2d6c7b5ec6" : {
+                        "storage" : {
+                            "0x00" : "0x01"
+                        }
+                    }
+                }
+            }
+        ],
+        "pre" : {
+            "0x0aabbccdd5c57f15886f9b263e2f6d2d6c7b5ec6" : {
+				"balance" : "0x00",
+                "code" : "{ (seq (MSTORE 0 0x0000111122223333444455556666777788889999aaaabbbbccccddddeeeeffff) (RETURN 0 32)) }",
```

Comment:
do ve have a test where return data is checked when the call fails on other reasons? 
like with new REVERT? 

or where this subcall has another subcall that sets return data but then the first subcall fails 
and thus all first subcall should be reverted and in this scenario we should check the return data in the original call.

Reply:
I'll add these to the spreadsheet, and write a comment here again.

Reply:
Case 1. https://docs.google.com/spreadsheets/d/1xat7UI8GtB4ZGVdlK5_XQSHJZaMThi4SrlcL8XMZb5Q/edit#gid=1605675414&range=C36

Reply:
I didn't understand the second case you described.

Reply:
TRANSACTION -> CALL -> (CALL -> CALL RETURN | OOG) | check return data

-> mean call
| finish call and go one level up